docs/05.PORT-API.md

```diff
@@ -151,6 +151,32 @@ jerry_port_get_native_module (jerry_value_t name) /**< module specifier */
 }
 ```
 
+## Promise
+
+```c
+/**
+ * HostPromiseRejectionTracker operations
+ */
+typedef enum
+{
+  JERRY_PROMISE_REJECTION_TRACKER_REJECT, /**< promise is rejected without any handlers */
+  JERRY_PROMISE_REJECTION_TRACKER_HANDLE, /**< handler is added to a rejected promise for the first time */
+} jerry_promise_rejection_tracker_t;
+
+/**
+ * Track unhandled promise rejections.
+ *
+ * Note:
+ *      This port function is called by jerry-core when JERRY_BUILTIN_PROMISE
+ *      is enabled.
+ *
+ * @param promise rejected promise
+ * @param operation HostPromiseRejectionTracker operation
+ */
+void jerry_port_track_promise_rejection (const jerry_value_t promise,
+                                         const jerry_promise_rejection_tracker_t operation);
+```
```

Comment:
BTW, shouldn't the "How to port JerryScript" section be extended?

Reply:
I've added it.